akka-stream/src/main/resources/reference.conf

```diff
@@ -45,17 +45,22 @@ akka {
       # this may cause an initial runtime overhead, but most of the time fusing is
       # desirable since it reduces the number of Actors that are created.
       auto-fusing = on
-      
+
       # Those stream elements which have explicit buffers (like mapAsync, mapAsyncUnordered,
       # buffer, flatMapMerge, Source.actorRef, Source.queue, etc.) will preallocate a fixed
       # buffer upon stream materialization if the requested buffer size is less than this
       # configuration parameter. The default is very high because failing early is better
       # than failing under load.
       #
-      # Buffers sized larger than this will dynamically grow/shrink and consume more memory 
+      # Buffers sized larger than this will dynamically grow/shrink and consume more memory
       # per element than the fixed size buffers.
       max-fixed-buffer-size = 1000000000
 
+      # Maximum number of sync messages that actor can process for stream to substream communication.
```

Comment:
That is a different thing. This is related to how many internal messages it processes *per actor message*. If you have your throughput at 1000, you probably don't want to set this to 1000. I am not sure why this new configuration setting is needed though.